doio.c

```diff
@@ -1811,7 +1811,24 @@ S_argvout_final(pTHX_ MAGIC *mg, IO *io, bool not_implicit) {
     return retval;
 }
 
-/* explicit renamed to avoid C++ conflict    -- kja */
+/*
+=for apidoc do_close
+
+Close an I/O stream.  This implements Perl L<perlfunc/C<close>>.
+
+C<gv> is the glob associated with the stream.
+
```

Comment:
We can't name the argument `explicit` in the source because that's a C++ keyword, but it's probably helpful to name it that in the documentation because it's less confusing.